ipn/ipnlocal/serve.go

```diff
@@ -56,41 +56,52 @@ type serveHTTPContext struct {
 	DestPort uint16
 }
 
-// serveListener is the state of host-level net.Listen for a specific (Tailscale IP, serve port)
+// localListener is the state of host-level net.Listen for a specific (Tailscale IP, serve port)
 // combination. If there are two TailscaleIPs (v4 and v6) and three ports being served,
 // then there will be six of these active and looping in their Run method.
 //
 // This is not used in userspace-networking mode.
 //
-// Most serve traffic is intercepted by netstack. This exists purely for connections
+// Most inbound traffic is intercepted by netstack. This exists purely for connections
```

Comment:
this sentence is a little confusing now. I know what it's trying to say, but only from a a lot of context and history. A future reader will just read it literally and most inbound traffic is not actually intercepted by netstack.

In fact, is most of the webclient traffic intercepted by netstack? I'd guess most web client traffic is local to the machine, people manipulating the host they're on, no?

Reply:
I tried rewriting this bit... see if that's clearer.

Reply:
oh, I was also going to reply specifically to your last question.  We don't really know how all the web client will be used, but initially we're certainly focused on more remote administration use cases like NAS devices.  We certainly want it to be usable for local administration (particularly linux), but we don't know how common that will be.